src/firmware/main/app/world/firmware_robot.h

```diff
@@ -9,30 +9,75 @@
  */
 typedef struct FirmwareRobot FirmwareRobot_t;
 
+/**
+ * This struct represents robot constants
+ */
+typedef struct RobotConstants
+{
+    // The mass of the entire robot
+    float mass;
+    // The moment of inertia of the entire robot
+    float moment_of_inertia;
+    // The maximum radius of the robot
+    float robot_radius;
+    // The maximum jerk this robot may safely undergo
+    float jerk_limit;
```

Comment:
Maybe throw in units here, unless they are defined somewhere else closeby explicitly.

Reply:
Nope, good catch, added units!